packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -3013,8 +3012,12 @@
   "Retail": "Retail",
   "Retention_setting_changed_successfully": "Retention policy setting changed successfully",
   "RetentionPolicy": "Retention Policy",
+  "RetentionPolicy_Advanced_Precision": "Use Advanced Retention Policy configuration",
+  "RetentionPolicy_Advanced_Precision_Cron": "Use Advanced Retention Policy Cron",
+  "RetentionPolicy_Advanced_Precision_Cron_Description": "How often the prune timer should run defined by cron job expression. Setting this to a more precise value makes channels with fast retention timers work better, but might cost extra processing power on large communities.",
```

Comment:
need to add a comment regarding the format (that includes seconds as the first value) and probably mention the documentation of the lib: http://bunkat.github.io/later/parsers.html#cron